addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,481 @@
+        run : function (actions) {
+            $('#country_id option:eq(1)').attr('selected', true);
+        }
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Select `Wire Transfer` payment method",
+        trigger: '#payment_method label:contains("Wire Transfer")',
+    },
+    {
+        content: "Pay Now",
+        extra_trigger: '#payment_method label:contains("Wire Transfer") input:checked,#payment_method:not(:has("input:radio:visible"))',
+        trigger: 'button[id="o_payment_form_pay"]:visible:not(:disabled)',
+    },
+    {
+        content: "Open Dropdown for See quotation",
+        extra_trigger: '.oe_cart .oe_website_sale_tx_status',
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "My account",
+        extra_trigger: '#top_menu li.dropdown .js_usermenu.show',
+        trigger: '#top_menu .dropdown-menu a[href="/my/home"]:visible',
+    },
+    {
+        content: "See Quotations",
+        trigger: '.o_portal_docs a:contains("Quotations")',
+    },
+
+    // enable extra step on website checkout and check extra step on checkout process
+    {
+        content: "Open Dropdown for logout",
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "Logout",
+        trigger: '#o_logout:contains("Logout")',
+    },
+    {
+        content: "Sign in as admin",
+        trigger: '#top_menu li a b:contains("Sign in")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function () {
+            $('.oe_login_form input[name="login"]').val("admin");
+            $('.oe_login_form input[name="password"]').val("admin");
+            $('.oe_login_form input[name="redirect"]').val("/shop/cart");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Open Customize menu",
+        trigger: '.o_menu_sections a:contains("Customize")',
+    },
+    {
+        content: "Enable Extra step",
+        trigger: 'a.dropdown-item label:contains("Extra Step Option")',
+    },
+    {
+        content: "Open Dropdown for logout",
+        extra_trigger: '.progress-wizard-step:contains("Extra Info")',
+        trigger: '#top_menu li.dropdown:visible a:contains("Mitchell Admin")',
+    },
+    {
+        content: "Logout",
+        trigger: '#o_logout:contains("Logout")',
+    },
+    {
+        content: "Sign in as abc",
+        trigger: '#top_menu li a b:contains("Sign in")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function () {
+            $('.oe_login_form input[name="login"]').val("abc@odoo.com");
+            $('.oe_login_form input[name="password"]').val("1admin@admin");
+            $('.oe_login_form input[name="redirect"]').val("/shop");
+            $('.oe_login_form').submit();
```

Comment:
You can also set `/shop?search=Storage Box` inside `redirect` [input](https://github.com/odoo-dev/odoo/blame/364b93d35bbc5b1970cf764cacd3a8d0306d7bfb/addons/web/views/webclient_templates.xml#L453) and remove next step.

Reply:
👍